mumd/src/state/server.rs

```diff
@@ -88,6 +89,44 @@ impl Server {
         &self.channels
     }
 
+    /// Takes a channel name and returns either a tuple with the channel id and a reference to the
+    /// channel struct if the channel name unambiguosly refers to a channel, or an error describing
+    /// if the channel identifier was ambigous or invalid.
+    /*/// note that doctests currently aren't run in binary crates yet (see #50784)
```

Comment:
What's with the `/*///` here?

Reply:
I did it as to not show it in the documentation and so it doesn't run. I can re-show it in the documentation though.